libs/csm/rest/csm_rest_iamuser.py

```diff
@@ -662,3 +662,43 @@ def verify_create_iam_user_rgw(
             self.log.error("Status code check failed.")
             result = False
         return result, resp
+
+    @RestTestLib.authenticate_and_login
+    def modify_iam_user_rgw(self, uid, payload: dict):
+        """
+        Modify IAM User parameters.
+        :param uid: userid
+        :param payload: payload for user creation
+        :return: response
+        """
+        self.log.info("Creating IAM user request....")
```

Comment:
change this description and other also as per call

Reply:
okay

Reply:
done